apps/system/utils/kdbg_date.c

```diff
@@ -235,9 +289,10 @@ int kdbg_date(int argc, char **args)
 
 	if (argc == 1) {
 		ret = date_showtime();
-	} else if (!strcmp(args[1], "-s") && argc == 6) {
+	} else if (!strcmp(args[1], "-s") && argc == 9) {
```

Comment:
In this case, #if !(defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED)), argc is 6. But you modify this without any conditional.